osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNote.cs

```diff
@@ -54,24 +115,151 @@ public override Color4 AccentColour
                     return;
                 base.AccentColour = value;
 
-                headPiece.AccentColour = value;
+                tickContainer.Children.ForEach(t => t.AccentColour = value);
+
                 bodyPiece.AccentColour = value;
-                tailPiece.AccentColour = value;
+                head.AccentColour = value;
+                tail.AccentColour = value;
             }
         }
 
         protected override void UpdateState(ArmedState state)
         {
         }
 
-        protected override void Update()
+        /// <summary>
+        /// Handles key down events on the body of the hold note.
+        /// </summary>
+        /// <param name="state">The input state.</param>
+        /// <param name="args">The key down args.</param>
+        /// <returns>Whether the key press was handled.</returns>

```

Comment:
Remove the xmldoc and add an inline comment explaining what it's doing IMO.

See my other comments.